CHANGES.md

```diff
@@ -36,6 +36,14 @@ OpenSSL 3.4
 
    *Stephan Wurm*
 
+ * OpenSSL libraries no longer use atexit(3) to perform cleanup.
+   The cleanup now runs automatically on behalf of library destructor.
```

Comment:
`on behalf of` sounds awkward to me in this context. I suggest: `... runs automatically from a shared library destructor of libcrypto.`